app/models/comment.rb

```diff
@@ -0,0 +1,13 @@
+class Comment < ActiveRecord::Base
+  belongs_to :user, class_name: 'User'
```

Comment:
- [ ] As stated by the previous reviewers, please make sure to have the association to a 'author' and the class_name: 'User` as it is, just like you did in the Post model. 🙏